### DISCUSSION TITLE
Delete elastic search logstash indices older than ...

### DISCUSSION THREAD

**Discussion by Rhoenit** [telekom-security/tpotce#1094] [General]
Normaly we set our tpots to delete elasticsearch indices older than 45 days (standard 90 days). 
I installed an new tpot and now I can´t find this in the new Version 22.04.0. 
What is the workaround?

**Comment by t3chn0m4g3** [comment]
Log persistence is described here: https://github.com/telekom-security/tpotce#log-persistence
